articles/evaluation_queues.md

```diff
@@ -0,0 +1,124 @@
+---
+title: Evaluation Queues
+layout: article
+excerpt: Evaluations queues are designed to support open-access data analysis and modeling challenges in Synapse. 
+category: howto
+---
+
+# Evaluation Queues
+Evaluations queues are designed to support open-access data analysis and modeling challenges in Synapse. This framework provides tools for administrators to collect and analyze data models from Synapse users created for a specific goal or purpose.
+
+## Create an Evaluation Queue 
+
+Synapse users can submit Synapse Files, or Synapse Docker containers to an Evaluation queue ("Evaluation" for short) which you manage.  An Evaluation has a "contentSource" field which during creation should be set the Synapse ID of your challenge project.  Afterwards your project will have a **Challenge** tab next to the Wiki, Files and Tables tabs.  Your new Evaluation(s) will appear in a list.   You can create a new Evaluation using the R, Python or web client.  There are also ways to define submission 'rounds' (start date, round duration, and number of rounds) with optional submission quota (maximum submissions per participant or team) for each Evaluation queue. 
```

Comment:
can submit any Synapse entity to a queue

Reply:
seems backwards to suggest to create an evaluation queue before having a challenge set up.

Reply:
can you have a queue without a challenge?

Reply:
a lot of content that describes what happens after creating - should be after the steps that actually do it (the code block below)

Reply:
No need to call out all clients here, clear from context below (might only mention if one of the clients didn't support it)

Reply:
too many details about rounds and configuration here.

Reply:
You can physically have a queue without a challenge, but I'll have to check if you can submit to it at all without having the project being a "challenge"